features/car/traffic_light_penalties.feature

```diff
@@ -0,0 +1,27 @@
+@routing @car @traffic_light
+Feature: Car - Handle traffic lights
+
+    Background:
+        Given the profile "car"
+
+    Scenario: Car - Encounters a traffic light
+        Given the node map
+            """
+            a b c 1 e 2 f
+                d
```

Comment:
Can you in addition add a test for a traffic light in the profile's turn branch?